controller/DeliveryRunner.php

```diff
@@ -36,11 +36,21 @@
  */
 class DeliveryRunner extends taoDelivery_actions_DeliveryServer
```

Comment:
this class doesn't exist anymore, should it be ``oat\taoDelivery\controller\DeliveryServer`` ?